apps/server/src/modules/files-storage-client/files-storage-client.module.ts

```diff
@@ -7,7 +8,7 @@ import { FilesStorageProducer } from './service/files-storage.producer';
 @Module({
 	imports: [LoggerModule],
 	controllers: [],
-	providers: [FilesStorageClientAdapterService, CopyFilesService, FilesStorageProducer],
+	providers: [FilesStorageClientAdapterService, CopyFilesService, FilesStorageProducer, CopyHelperService],
```

Comment:
CopyHelperService - i am not sure but it should not exist any needs for it or?

Reply:
Fixed in [BC-2417 - copy module structure](https://github.com/hpi-schul-cloud/schulcloud-server/pull/3832/files#top)
#3832